src/main/java/techreborn/blockentity/machine/multiblock/FusionControlComputerBlockEntity.java

```diff
@@ -237,19 +211,19 @@ public void tick() {
 			}
 		}
 
-		if(lastTick == world.getTime()){
+		if (lastTick == world.getTime()) {
 			//Prevent tick accerators, blame obstinate for this.
 			return;
 		}
 		lastTick = world.getTime();
 
 		// Force check every second
 		if (world.getTime() % 20 == 0) {
-			checkCoils();
+			isMultiblockValid();
 			inventory.setChanged();
 		}
 
-		if (coilCount == 0) {
+		if (!isMultiblockValid()) {
```

Comment:
Checking multiblockvalid every tick, voiding the 

`		if (world.getTime() % 20 == 0) {
			checkCoils();
			isMultiblockValid();
			inventory.setChanged();
		}
`

above it.